webanno-ui-annotation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/annotation/dialog/OpenDocumentDialogPanel.java

```diff
@@ -201,6 +221,78 @@ protected void onEvent(final AjaxRequestTarget aTarget)
 
         private Project projectSelection;
         private SourceDocument documentSelection;
+        private User userSelection;
+    }
+    
+    private class UserSelectionForm
```

Comment:
We have `de.tudarmstadt.ukp.clarin.webanno.ui.core.users.UserSelectionPanel` - maybe that could be directly re-used here instead of implemeting a new selection panel for users? Or at least `UserSelectionPanel` may illustrate how to implement a selection list with fewer lines of code.